### PR TITLE
py-basalt: new version v0.2.2 promoted to BB5

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

**Comment by tristan0x** [comment]
Is is normal that CI builds coreneuron, neurodamus, ...  @matz-e ?

**Comment by matz-e** [comment]
They failed upstream thanks to an `LC_ALL=....utf-8` and gcc behavior. Building is fine, build log parsing is not. Subsequent CI runs should be fine.

Retest this please.

File: var/spack/repos/builtin/packages/py-basalt/package.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class PyBasalt(PythonPackage):
     url      = "git@github.com:tristan0x/basalt.git"
     
     version('develop', git=url, branch='master', submodules=True, clean=False)
+    version('0.2.2', git=url, tag='v0.2.2', submodules=True, preferred=True, clean=False)
     version('0.2.1', git=url, tag='v0.2.1', submodules=True, preferred=True, clean=False)
     version('0.1.1', git=url, tag='v0.1.1', submodules=True, preferred=True, clean=False)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-basalt/package.py]:
```suggestion
    version('0.1.1', git=url, tag='v0.1.1', submodules=True, clean=False)
```

File: var/spack/repos/builtin/packages/py-basalt/package.py [hunk 2]
```diff
@@ -13,6 +13,7 @@ class PyBasalt(PythonPackage):
     url      = "git@github.com:tristan0x/basalt.git"
     
     version('develop', git=url, branch='master', submodules=True, clean=False)
+    version('0.2.2', git=url, tag='v0.2.2', submodules=True, preferred=True, clean=False)
     version('0.2.1', git=url, tag='v0.2.1', submodules=True, preferred=True, clean=False)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-basalt/package.py]:
```suggestion
    version('0.2.1', git=url, tag='v0.2.1', submodules=True, clean=False)
```

**Review by matz-e — CHANGES_REQUESTED** [review]
LGTM apart from preferences.

**Review by matz-e — APPROVED** [review]

**Pull request merged**